fastlane/Fastfile

```diff
@@ -202,31 +207,52 @@ platform :android do
         end
 
 
+        private_lane :determine_release_notes do |options|
+            notes_type = options[:notes_type]
+            custom_release_notes = options[:release_notes]
```

Comment:
if you provide custom release notes, it would be nice if that automatically set the `notes_type` to custom